src/js/App/Header/Beta.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tooltip, Button } from '@patternfly/react-core';
+import cookie from 'js-cookie';
+
+const Beta = () => (
+    <div className={'ins-c-page__beta ' + (window.location.href.includes('/beta') ? '' : 'ins-c-page_beta-hidden')}>
```

Comment:
Can you do conditional rendering here?

just check to see if it's /beta and render it instead of setting a classname that will hide something.